vars/cloudFoundryDeploy.groovy

```diff
@@ -11,6 +11,11 @@ import static com.sap.piper.Prerequisites.checkScript
 @Field String STEP_NAME = getClass().getName()
 
 @Field Set GENERAL_CONFIG_KEYS = [
+    /**
+     * Defines the tool used for the build.
```

Comment:
I would maybe remove the documentation for this as it is at the end only a technical implementation detail as users should rather set the deployTool instead the buildTool or add a sentence explain that.

Reply:
Sure I'll address that.